api/src/pcapi/routes/public/individual_offers/v1/serialization.py

```diff
@@ -804,8 +804,14 @@ def validate_ids_at_provider(cls, ids_at_provider: str) -> list[str] | None:
         return None
 
 
-class GetDatesQueryParams(IndexPaginationQueryParams):
-    pass
+class GetEventStocksQueryParams(IndexPaginationQueryParams):
+    ids_at_provider: str | None = fields.IDS_AT_PROVIDER_FILTER
+
+    @pydantic_v1.validator("ids_at_provider")
+    def validate_ids_at_provider(cls, ids_at_provider: str) -> list[str] | None:
+        if ids_at_provider:
+            return ids_at_provider.split(",")
```

Comment:
```suggestion
            return [item.strip() for item in ids_at_provider.split(",")]
```

Avec un petit `strip()` sur chaque élément ? Parce que ça ne coûte pas grand chose et que pourra nous éviter des questions de support ? :)